cmake/opentelemetry-proto.cmake

```diff
@@ -84,7 +84,12 @@ foreach(IMPORT_DIR ${PROTOBUF_IMPORT_DIRS})
   list(APPEND PROTOBUF_INCLUDE_FLAGS "-I${IMPORT_DIR}")
 endforeach()
 
-set(gRPC_CPP_PLUGIN_EXECUTABLE $<TARGET_FILE:gRPC::grpc_cpp_plugin>)
+
+if(CMAKE_CROSSCOMPILING)
+    find_program(gRPC_CPP_PLUGIN_EXECUTABLE grpc_cpp_plugin)
```

Comment:
Does this `find_program` only work if `grpc_cpp_plugin` is available in system search path?

Reply:
yes, but you can specify your own directory through -DCMAKE_PROGRAM_PATH=/your/path